src/pybind/mgr/balancer/module.py

```diff
@@ -225,6 +225,8 @@ class Module(MgrModule):
             'name': 'begin_weekday',
             'type': 'uint',
             'default': 0,
+            'min': '0',
```

Comment:
I'm not sure, but i think you can use an integer value here. At least we set them in the [Dashboard](https://github.com/ceph/ceph/blob/master/src/pybind/mgr/dashboard/module.py#L251). You can also use the [Option](https://github.com/ceph/ceph/blob/master/src/pybind/mgr/mgr_module.py#L355) class to define your module options.